.github/workflows/integration.yaml

```diff
@@ -32,6 +32,10 @@ env:
   CARGO_NET_RETRY: 10
   RUSTUP_MAX_RETRIES: 10
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
```

Comment:
There is more going on here than just denying clippy warnings.

Can you describe what this change does?

Reply:
Updated the code + PR description.